web/src/components/Header.tsx

```diff
@@ -70,8 +71,10 @@ export const Header = ({ actionItems = [], logoHref, cityName, language, TabBar
       <Paper>
         <HeaderContainer ref={ref}>
           <Row>
-            <HeaderLogo link={logoHref} />
-            {!!cityName && <HeaderTitle title={cityName} landingPath={landingPath} />}
+            <Stack direction='row-reverse' gap={1}>
```

Comment:
Can anyone tell me why I need to set `row-reverse` here? 😨  I don't get it 🙈

Reply:
shouldn't it be used within conditional, if it is rtl then row-reverse? :thinking:

Reply:
In theory it should not be necessary at all since this is usually automatically handled by mui. But its actually necessary for both ltr and rtl.

Reply:
I think it has to do with the order at the `HeaderLogo` but your solution is ok.